odpc.client/src/router/index.ts

```diff
@@ -53,8 +63,12 @@ const router = createRouter({
 router.beforeEach(async (to) => {
   document.title = `${to.meta?.title || ""} | ${import.meta.env.VITE_APP_TITLE}`;
 
+  document.body.setAttribute("tabindex", "-1");
+  document.body.focus();
+  document.body.removeAttribute("tabindex");
+
   const requiresAuth = to.matched.some((route) => route.meta.requiresAuth);
-  const user = await getUser(false);
+  const user = await getUser(true);
```

Comment:
Is het wellicht veiliger met het verlopen van de sessie om deze waarde nooit te cachen?

Reply:
Yes, done